be/src/service/service_be/lake_service.cpp

```diff
@@ -429,4 +435,245 @@ void LakeServiceImpl::unlock_tablet_metadata(::google::protobuf::RpcController*
     }
 }
 
+Status _get_existing_files_from_remote(BrokerServiceConnection& client, const std::string& remote_path,
```

Comment:
compile unit only function should be declared as `static` or move into anonymous namespace to avoid unexpected symbol pollution.